x-pack/packages/kbn-elastic-assistant/impl/assistant/types.ts

```diff
@@ -15,3 +15,10 @@ export interface Prompt {
   isDefault?: boolean;
   isNewConversationDefault?: boolean;
 }
+
+export interface SelectedPromptContext {
```

Comment:
nit: Maybe live next to `PromptContext` over in `x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_context/types.ts`?